pom.xml

```diff
@@ -4,8 +4,9 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.microsoft.office</groupId>
+    <groupId>com.microsoft.ewsjavaapi</groupId>
     <artifactId>ews-java-api</artifactId>
+    
     <version>1.3-SNAPSHOT</version>
```

Comment:
I suspect that by now we have broken the public API compared to 1.2, I wonder if we should move the version to 2.0-SNAPSHOT?  Is there a tool that compares the 1.2 vs. 1.3 and tell us if we have broken the library public interface?  I suspect we will break it before release.  Thoughts?